admin/pre_reg/utils.py

```diff
@@ -5,10 +5,14 @@
 
 
 def get_prereg_reviewers():
-    return MyUser.objects.filter(
-        groups__name='prereg_group'
-    ).annotate(
-        fuller_name=Concat('first_name', Value(' '), 'last_name')
-    ).values_list(
-        'email', 'fuller_name'
-    )
+    # Note - fixes django.db.utils.OperationalError: no such table' error if ever one erases a db table and tries to remigrate
```

Comment:
I think, if you moved the call for this from the form __init__ method to the form's get_initial method you wouldn't have this exception happen. Either way, the clause is too wide. You should trim it down to just OperationError. Lots of other things could go wrong in this statement and the wide exception handling would make it quite the mystery.